packages/react-native/src/components/3DS/ThreeDSecureFrame.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { useState } from "react";
+import { View, StyleSheet } from "react-native";
+import { useEvervault } from "../EvervaultProvider";
+import WebView from "react-native-webview";
+import { CHALLENGE_DOMAIN_3DS } from "./config";
+import { Spinner } from "./Spinner";
+import { defaultThreeDSecureFrameStyles as defaultStyles } from "./styles";
+
+interface ThreeDSecureFrameProps {
+  sessionId: string;
+}
+
+export function ThreeDSecureFrame({ sessionId }: ThreeDSecureFrameProps) {
```

Comment:
Rather than taking the sessionId as a prop it would be cleaner to pull it from the context of the `ThreeDSecure` provider.

```
const { sessionId } = useContenxt(ThreeDSecureContext)
```